frida_tools/application.py

```diff
@@ -445,6 +451,47 @@ def work():
 
         return result[0]
 
+    def return_real_args(self, parser):
```

Comment:
It seems like these helper functions don't need to access `self`, so they could be moved outside the class (below it).

Reply:
Good catch -- done!